.gitignore

```diff
@@ -87,3 +88,6 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
+
+.vscode
+.vs
```

Comment:
Unneeded change. Please add new line at the end of file and configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).